### PR TITLE
DOC-8379: Note on avoiding task overlap in backup service

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/learn/pages/services-and-indexes/services/backup-service.adoc [hunk 1]
```diff
@@ -83,6 +83,14 @@ Optionally, the data itself can be retained, on the local filesystem.
 A deleted repository whose data still exists can be _imported_ back into the cluster, if required.
 Once imported, the repository can be _read_ from, but no longer receives data backups.
 
+[#avoiding-task-overlap]
+== Avoiding Task Overlap
+
+Although the Backup Service allows automated tasks to be scheduled for repetition at intervals as small as one minute, administrators are recommended typically not to lower the interval below fifteen minutes; and always to ensure that the established interval allows each scheduled task-repetition ample time to complete before the next; even in the event of unanticipated, high network-latency.
+
+If, due to an interval-specification that is too small, a given task attempts to start while its prior repetition is still ongoing, the new task fails, stops running, and is not restarted.
```
Comment (owend74) [inline modules/learn/pages/services-and-indexes/services/backup-service.adoc]: If, due to an interval-specification that is too small, a given task attempts to start while its prior repetition is still ongoing, the new task fails to start.

File: modules/learn/pages/services-and-indexes/services/backup-service.adoc [hunk 2]
```diff
@@ -83,6 +83,14 @@ Optionally, the data itself can be retained, on the local filesystem.
...
+In such circumstances, all data continues to be backed up by the prior repetition; and will be backed up again by any subsequent repetition that starts after the completion of the prior repetition.
```
Comment (owend74) [inline modules/learn/pages/services-and-indexes/services/backup-service.adoc]: In such circumstances, all data continues to be backed up by the prior repetition; and will be backed up again by any subsequent repetition that starts after the completion of the previous backup.

**Review by owend74 — CHANGES_REQUESTED** [review]
I got a little lost in repetitions :)
Not sure I improved - so treat just suggestions..

@cg14823 probably worth you reading through.

File: modules/learn/pages/services-and-indexes/services/backup-service.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/learn/pages/services-and-indexes/services/backup-service.adoc]: ok.

File: modules/learn/pages/services-and-indexes/services/backup-service.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/learn/pages/services-and-indexes/services/backup-service.adoc]: ok

File: modules/learn/pages/services-and-indexes/services/backup-service.adoc [hunk 3]
```diff
@@ -83,6 +83,14 @@ Optionally, the data itself can be retained, on the local filesystem.
 A deleted repository whose data still exists can be _imported_ back into the cluster, if required.
 Once imported, the repository can be _read_ from, but no longer receives data backups.
 
+[#avoiding-task-overlap]
+== Avoiding Task Overlap
+
+Although the Backup Service allows automated tasks to be scheduled for repetition at intervals as small as one minute, administrators are recommended typically not to lower the interval below fifteen minutes; and always to ensure that the interval is large enough to allow each scheduled task-repetition ample time to complete before the next is due to commence; even in the event of unanticipated network latency.
```
Comment (cg14823) [inline modules/learn/pages/services-and-indexes/services/backup-service.adoc]: I think `repetition` may be a bit confusing in some of the places I would suggest altering the following

> Backup Service allows automated tasks to be scheduled for repetition at intervals as small as one minute

To

> Backup Service allows automated task to be scheduled at intervals as small as one minutes

**Review by cg14823 — COMMENTED** [review]
I think it looks good but maybe the word repetition can be a bit confusing in some context maybe iteration or prior instance could be replaced in some of the places.

File: modules/learn/pages/services-and-indexes/services/backup-service.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/learn/pages/services-and-indexes/services/backup-service.adoc]: The reason I used 'repetition' is that I imagined we are talking about the same task, scheduled to be repeated at intervals?.But perhaps that is not correct. If *different* tasks overlap, does this also produce failure of this kind?

**Review by cg14823 — APPROVED** [review]

**Pull request merged**